### PR TITLE
new patches.vizor.io homepage

### PR DESCRIPTION
**Pull request by gmarinov**
Landing page updates to Patches site. Minor updates to Editor and internal pages visuals. 

Tested on desktop: Safari, Chrome, Firefox, mobile: iOS iPhone/iPad, Android Samsung Galaxy S6, S8 Chrome and built-in Internet browser.

screen shot 2017-07-13 at 09 49 03p

screen shot 2017-07-13 at 09 48 52p

screen shot 2017-07-13 at 09 49 13

screen shot 2017-07-17 at 09 12 10

### CODE REVIEW

**Comment by esaruoho** [comment]
@gmarinov says: the homepage will need retagging **8** experiences with "frontpage" when it's deployed

File: app.js [hunk 1]
```diff
@@ -278,6 +278,8 @@ app.get('/', function(req, res, next) {
 	}
 })
 
+app.get('/v1', homeController.index.bind(homeController))
```
Comment (kschzt) [inline app.js]: what's this route for?

Comment (gmarinov) [inline app.js]: testing, without affecting `/`. needs removing upon deployment (patches.vizor.io).

Comment (kschzt) [inline app.js]: please remove it

**Pull request merged**